flask_admin/templates/bootstrap4/admin/model/modals/details.html

```diff
@@ -35,6 +35,4 @@ <h3>{{ _gettext('View Record') + ' #' + request.args.get('id') }}</h3>
 {% endblock %}
 
 {% block tail %}
-  <script src="{{ admin_static.url(filename='admin/js/details_filter.js', v='1.0.0') }}"></script>
```

Comment:
Great work!

Why deleting `details_filter.js`?

Reply:
My mistake, I just returned it back.